lib/screens/screen.dart

```diff
@@ -0,0 +1,24 @@
+import 'dart:io';
+
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/widgets.dart';
+import 'package:survey/core/environment.dart';
+
+class Screen extends StatelessWidget {
+  final Widget body;
+  Screen({this.body});
+
+  @override
+  Widget build(BuildContext context) {
+    if (Platform.isAndroid) {
+      return Scaffold(
+        body: body,
+      );
+    }
+
+    return CupertinoPageScaffold(
```

Comment:
In scope of IC, I will assume that we only run project on Android and iOS 😆  I was lazy to open simulator, so I opened project by Chrome and crash happened 🤣